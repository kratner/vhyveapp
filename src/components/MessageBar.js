import React from "react";
import styled from "styled-components";

const MessageContainer = styled.div`
  position: absolute;
  top: 0;
  z-index: 5;
  color: #fff;
  font-size: 0.5em;
  text-align: center;
  left: 50%;
  transform: translateX(-50%);
  width: 100%;
  @media only screen and (orientation: portrait) {
    padding-top: 1.5em;
    top: 50%;
    transform: translateY(-50%);
    transform: rotate(90deg);
  }
`;

export default props => {
  return <MessageContainer id={props.id}>{props.message}</MessageContainer>;
};
