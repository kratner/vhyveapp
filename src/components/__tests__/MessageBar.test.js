import React from "react";
import { shallow } from "enzyme";
import MessageBar from "../MessageBar";

describe("MessageBar", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<MessageBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
