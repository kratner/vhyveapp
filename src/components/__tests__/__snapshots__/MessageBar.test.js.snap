// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageBar renders correctly 1`] = `ShallowWrapper {}`;
